plugins/module_utils/acme/acme.py

```diff
@@ -52,18 +52,22 @@
     IPADDRESS_IMPORT_ERROR = None
 
 
+RETRY_STATUS_CODES = (408, 429, 503)
```

Comment:
This could be more readable if it used names
```python
from http import HTTPStatus

RETRY_STATUS_CODES = {
    HTTPStatus.BAD_REQUEST,
    HTTPStatus.REQUEST_TIMEOUT,
    HTTPStatus.SERVICE_UNAVAILABLE,
}
```
These enums are comparable with regular numbers too:
```pycon
In [36]: from http import HTTPStatus

In [37]: 400 in {HTTPStatus.BAD_REQUEST, HTTPStatus.REQUEST_TIMEOUT, HTTPStatus.SERVICE_UNAVAILABLE}
Out[37]: True
```

Reply:
That's Python 3.5+, so it won't work with Python 2.6 or Python 2.7 (both which are still supported by this collection).